.factory/automation.yml

```diff
@@ -60,6 +60,7 @@ build:
       image: vaticle-ubuntu-22.04
       command: |
         bazel test //rust/... --test_output=errors
+        bazel test //rust/tests/behaviour/... --test_output=errors
```

Comment:
`//rust/tests/behaviour/...` is a subset of `//rust/...`, so we shouldn't have both lines here.

To be honest, I'm not entirely sure why the `test-java` job is so specific either. It could just run `bazel test //java/...`.